src/workerd/api/html-rewriter.c++

```diff
@@ -1063,11 +1063,10 @@ jsg::Ref<Response> HTMLRewriter::transform(jsg::Lock& js, jsg::Ref<Response> res
 
   auto& ioContext = IoContext::current();
 
-  // lol-html writes to a pipe, the other end of which is our transformed response body.
-  auto ts = IdentityTransformStream::constructor(js);
-  response = Response::constructor(js, kj::Maybe(ts->getReadable()), kj::mv(response));
-
-  auto outputSink = ts->getWritable()->removeSink(js);
```

Comment:
For reviewers: If you don't have the context on this change, what is happening in the original code here is that we allocate an `IdentityTransformStream`, which creates an internal refcounted `IdentityTransformStreamImpl` instance, then uses that to create one `WritableStream` instance and one `ReadableStream` instance. We then immediately extract the reference to the `IdentityTransformStreamImpl` out of the `WritableStream`. In this case, creating both the `IdentityTransformStream` and the `WritableStream` are unnecessary since both are immediately thrown away.

In the new code, we create only the `IdentityTransformStreamImpl` and the `ReadableStream` that we are keeping.